### DISCUSSION TITLE
Log Postgres

### DISCUSSION THREAD

**Discussion by legithubdeaymeric** [PostgREST/postgrest#1892] [Q&A] [answered]
Hello,

It is not an issue but a question, the answer may be simple.
We want to have in the posgres logs, the role used by posgrest, not the anonymous role.

We use a custom role with a JWT Web token to authenticate to the API.

Is it possible?

**Comment by steve-chavez** [comment, marked as answer]
Hey @legithubdeaymeric,

> We want to have in the posgres logs, the role used by posgrest, not the anonymous role.

Hm, could you show a sample of how your logs look like? I believe they should already include a `set local role = '<jwt_role>'`.

Additionally, if you want to identify PostgREST inside pg logs, you could use an `application_name` on the connection string. Like:

```
db-uri="postgres://user@localhost/mydb?application_name=pgrest"
```
